web/src/main/java/com/futurewei/alcor/web/entity/dataplane/InternalPortEntityNB.java

```diff
@@ -0,0 +1,57 @@
+package com.futurewei.alcor.web.entity.dataplane;
```

Comment:
For every new files, please configure your IDE to automatically insert our license statement. 

An example: https://github.com/futurewei-cloud/alcor/blob/master/web/src/main/java/com/futurewei/alcor/web/entity/port/PortEntity.java